public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -452,3 +474,23 @@ function useGetColumns(
     },
   ];
 }
+
+function UnusedContactPointBadge() {
+  const styles = useStyles2(unusedContactPointBadgeStyles);
+
+  return (
+    <Badge
+      text="Unused"
+      color="orange"
+      icon="exclamation-triangle"
+      tooltip="This contact point is not used in any notification policy and it will not receive any alerts"
+      className={styles.badge}
+    />
+  );
+}
+
+const unusedContactPointBadgeStyles = () => ({
+  badge: css`
+    cursor: default;
```

Comment:
I'm a bit surprised that this isn't the default cursor style?

Reply:
I reached out to the design system team, and the consensus is that the default cursor should be `text`. 
I'll remove this change